docs/core/install/linux-ubuntu.md

```diff
@@ -0,0 +1,364 @@
+---
+title: Install .NET Core on Ubuntu - .NET Core
+description: Demonstrates the various ways to install .NET Core SDK and .NET Core Runtime on Ubuntu.
+author: thraka
+ms.author: adegeo
+ms.date: 05/18/2020
+---
+
+# Install .NET Core SDK and .NET Core Runtime on Ubuntu
+
+.NET Core is supported on different Ubuntu versions. This article describes how to install .NET Core on Ubuntu with APT. When an Ubuntu version falls out of support, .NET Core is no longer supported with that version. However, these instructions may help you to get .NET Core running on those versions, even though it isn't supported.
+
+## Supported distributions
+
+The following is a list of currently supported .NET Core releases and the versions of Ubuntu they're supported on. These versions remain supported until either the version of [.NET Core reaches end-of-support](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) or the version of [Ubuntu reaches end-of-life](https://wiki.ubuntu.com/Releases).
+
+For the best compatibility, choose a long-term release (LTS) version.
+
+| .NET Core | Ubuntu version (LTS) | Ubuntu version |
+|-----------|----------------------|----------------|
+| 2.1 (LTS) | 16.04, 18.04, 20.04  | 19.10          |
```

Comment:
The "Ubuntu version" column isn't immediately obvious what it represents, and a little weird that it's different from how .NET Core LTS is represented. I'd also slightly favor not having a separate list so that 1. it's easier to find the "next-down" and 2. it's easier to find my version in these lists (naturally doing a pseudo-binary-search with my eyes). Throwing this out there (w/ 5.0 for demonstration):

| .NET Core | Ubuntu version |
|-----------|----------------|
| **2.1 (LTS)** | **16.04 (LTS)**, **18.04 (LTS)**, 19.10, **20.04 (LTS)** |
| **3.1 (LTS)** | **16.04 (LTS)**, **18.04 (LTS)**, 19.10, **20.04 (LTS)** |
| 5.0 | **16.04 (LTS)**, **18.04 (LTS)**, 19.10, **20.04 (LTS)** |

Reply:
Fixed

Reply:
I'm still not a fan of the out-of-orderness, but other than that looks better now to me.